github3/session.py

```diff
@@ -21,7 +21,7 @@ class GitHubSession(requests.Session):
     auth = None
     __attrs__ = requests.Session.__attrs__ + ['base_url', 'two_factor_auth_cb']
 
-    def __init__(self):
+    def __init__(self, isapp=False):
```

Comment:
This class shouldn't accept parameters. The way we should handle this is described above.